docs/tutorials/multimodal/beginner_image_cls.md

```diff
@@ -0,0 +1,88 @@
+# AutoMM for Image Classification: Quick Start
+:label:`sec_automm_imageclassification_beginner`
+
+In this quick start, we'll use the task of image classification to illustrate how to use `AutoMMPredictor`. Once the data is prepared in [Pandas DataFrame](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html) format, a single call to `AutoMMPredictor.fit()` will take care of the model training for you.
+
+
+## Create Image Dataset
+
+For demonstration purposes, we use a subset of the [Shopee-IET dataset](https://www.kaggle.com/c/shopee-iet-machine-learning-competition/data) from Kaggle.
+Each image in this data depicts a clothing item and the corresponding label specifies its clothing category.
+Our subset of the data contains the following possible labels: `BabyPants`, `BabyShirt`, `womencasualshoes`, `womenchiffontop`.
+
+We can load a dataset by downloading a url data automatically:
+
+```{.python .input}
+from autogluon.vision import ImageDataset
+train_dataset, _, test_dataset = ImageDataset.from_folders("https://autogluon.s3.amazonaws.com/datasets/shopee-iet.zip")
+print(train_dataset)
+```
+
+We can see there are 800 rows and 2 columns in this training dataframe. The 2 columns are `image` and `label`, and each row represents a different training sample
+
+
+## Use AutoMM to Fit Models
+
+Now, we fit a classifier using AutoMM as follows:
+
+```{.python .input}
+from autogluon.multimodal import AutoMMPredictor
+predictor = AutoMMPredictor(label="label", path="./automm_imgcls")
+predictor.fit(
+    train_data=train_dataset,
+    time_limit=30, # seconds
+) # you can trust the default config, e.g., we use a `swin_tiny_patch4_window7_224` model
```

Comment:
We use `swin_base_patch4_window7_224` as the default.

Reply:
Just changed